simphony_ui/ui.py

```diff
@@ -0,0 +1,199 @@
+from traits.api import (HasStrictTraits, Int, Float, Enum, Instance,
+                        Str, Directory, BaseInt, BaseFloat, Array, File)
+from traitsui.api import View, Item, UItem, Tabbed, VGroup, HGroup
+import os
+
+
+class PositiveFloat(BaseFloat):
+    """ A custom trait representing a positive float """
+
+    default_value = 1.0
+
+    info_text = 'a positive float'
+
+    def validate(self, object, name, value):
+        value = super(PositiveFloat, self).validate(object, name, value)
+        if value > 0.0:
+            return value
+
+        self.error(object, name, value)
+
+
+class PositiveInt(BaseInt):
+    """ A custom trait representing a positive integer """
+
+    default_value = 1
+
+    info_text = 'a positive integer'
+
+    def validate(self, object, name, value):
+        value = super(PositiveInt, self).validate(object, name, value)
+        if value > 0:
+            return value
+
+        self.error(object, name, value)
+
+
+class GlobalParametersModel(HasStrictTraits):
+    """ The model of Global input parameters """
+
+    num_iterations = Int(10)
+
+    force_type = Enum('Stokes', 'Coul', 'Dala')
+    force_type.default_value = 'Stokes'
+
+    traits_view = View(
+        VGroup(
+            Item(name='num_iterations', label='Number of iterations'),
+            Item(name='force_type'),
+        )
+    )
+
+
+class LiggghtsModel(HasStrictTraits):
+    """ The model of Liggghts input parameters """
+
+    timestep = Float(1e-6)
+    num_iterations = Int(10)
+
+    input_file = File()
```

Comment:
I would go for a path. File is an actual open file. I don't know how it will be rendered in the UI, but it's probably not what you want.